packages/devtools/lib/src/framework/framework.dart

```diff
@@ -39,15 +45,93 @@ class Framework {
 
   Screen current;
 
+  Screen previous;
+
   StatusLine globalStatus;
   StatusLine pageStatus;
   StatusLine auxiliaryStatus;
   ActionsContainer globalActions;
   ConnectDialog connectDialog;
+  ImportMessage importMessage;
   AnalyticsOptInDialog analyticsDialog;
 
   final Map<Screen, CoreElement> _screenContents = {};
 
+  void _initDragDrop() {
+    window.addEventListener('dragenter', (e) => _onDragEnter(e), false);
+    window.addEventListener('dragover', (e) => _onDragOver(e), false);
+    window.addEventListener('dragleave', (e) => _onDragLeave(e), false);
+    window.addEventListener('drop', (e) => _onDrop(e), false);
+  }
+
+  void _onDragEnter(MouseEvent event) {
+    final Element dropTarget = event.target;
+    dropTarget.classes.add('over');
+  }
+
+  void _onDragOver(MouseEvent event) {
+    // This is necessary to allow us to drop.
+    event.preventDefault();
+    event.dataTransfer.dropEffect = 'move';
+  }
+
+  void _onDragLeave(MouseEvent event) {
+    final Element dropTarget = event.target;
+    dropTarget.classes.remove('over');
```

Comment:
remove this logic to add and remove the 'over' css class as we aren't using it.
I expect if we did something similar we would want to make it more targeted visualizing dropping only into certain divs on the document. For example see the effects while dragging files into text entry areas on github.